samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/QuestionScoresBean.java

```diff
@@ -83,7 +83,7 @@
   private String groupName;
   @Getter @Setter
   private double maxScore;
-  @Getter @Setter
+  @Setter
   private List agents = null;
```

Comment:
Initialize the array list at the class member.

Reply:
@ern Changes done!